src/content/en/2021/privacy.md

```diff
@@ -212,76 +231,78 @@ Sensors can be useful to make a website more interactive, but could also be abus
 
 ### Media devices
 
-The MediaDevices API can be used to access connected media input such as cameras, microphones and screen sharing.
+The [MediaDevices API](https://developer.mozilla.org/en-US/docs/Web/API/MediaDevices) can be used to access connected media input such as cameras, microphones and screen sharing.
 
 {{ figure_markup(
-  image="nb_websites_with_mediadevices_blink_usage.png",
-  caption="Percentage of websites that request information to media devices",
-  description="Bar chart showing the percentage of websites that use the enumerateDevices() method to obtain information about the connected media devices.",
-  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vRfOwcNkLp-mYBkmhDV5AFxl8p0ls9oxFnmmo0WUcAJxjxQqmHjquRZCWj_lNZRyFtX3RdH5T92IESu/pubchart?oid=1398654768&format=interactive",
+  caption="Percent of desktop pages that used the `MediaDevicesEnumerateDevices` API.",
+  content="7.23%",
+  classes="big-number",
   sheets_gid="2141743069",
   sql_file="nb_websites_with_mediadevices_blink_usage.sql"
-  )
+)
 }}
 
-On 7.23% of websites, the `enumerateDevices()` method is called, which provides a list of the connected input devices.
+On 7.23% of desktop websites, and 5.33% of mobile websites the `enumerateDevices()` method is called, which provides a list of the connected input devices.
 
 ### Geolocation-as-a-service
 
-Geolocations services provide GPS and other location data of the user (such as <a hreflang="en" href="https://developer.mozilla.org/en-US/docs/Glossary/IP_Address">IP address</a>) and can be used by trackers to provide more relevant content to the user among other things.
-Therefore, we analyze the use of geolocation-as-a-service technologies on websites, based on libraries detected through [Wappalyzer](./methodology#wappalyzer).
+Geolocations services provide GPS and other location data of the user (such as [IP address](https://developer.mozilla.org/en-US/docs/Glossary/IP_Address)) and can be used by trackers to provide more relevant content to the user among other things. Therefore, we analyze the use of "geolocation-as-a-service" technologies on websites, based on libraries detected through [Wappalyzer](./methodology#wappalyzer).
```

Comment:
```suggestion
Geolocation services provide GPS and other location data (such as [IP address](https://developer.mozilla.org/en-US/docs/Glossary/IP_Address)) of the user and can be used by trackers to provide more relevant content to the user among other things. Therefore, we analyze the use of "geolocation-as-a-service" technologies on websites, based on libraries detected through [Wappalyzer](./methodology#wappalyzer).
```

Reply:
Happy for you to directly edit this branch and make these minor changes of things you've spotted in addition.

If reverting any of my suggested changes however could you add as comment instead to discuss?

Reply:
I didn't notice that I reverted a change! You moved the 'of the user' then?

Reply:
You didn't. I meant if you were going to.